src/participant/participant.ts

```diff
@@ -164,10 +165,10 @@ export default class ParticipantController {
   }
 
   async _getChatResponse({
-    messages,
+    prompt,
     token,
   }: {
-    messages: vscode.LanguageModelChatMessage[];
+    prompt: PromptResult;
```

Comment:
[nit] No action needed here, mostly thinking out loud. I wonder if we're over using the term `prompt` here. (or maybe it's in other places we could change it).
Is there a way we can name this to get the meaning quicker on first read? Maybe `ChatRequest`? `ChatCompletionRequest`?

Reply:
Also, it is a bit hard to get while reading, that prompt is an input argument but has PromptResult type. Like input is a result.

Reply:
Agree - wasn't quite happy with the name myself. I changed it to `ModelInput` - still don't think it's ideal, but I feel it's more descriptive than before.